### PR TITLE
feat: add new UI

### PR DESCRIPTION
**Pull request by MarioBatalha**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206412086603101

### CODE REVIEW

File: views/home/hero/index.tsx [hunk 1]
```diff
@@ -39,19 +40,29 @@ const HeroSection: FC = () => {
...
           >
-            Built on solana
-            <SolanaLogo maxHeight="1.5rem" maxWidth="1.5rem" />
+            <SolanaLogo maxHeight="2rem" maxWidth="2rem" />
+            <Typography
+              variant="small"
+              color="#000"
+              textAlign="center"
+              fontFamily="Roboto"
+              fontSize="14px"
```
Comment (KipandaJr) [inline views/home/hero/index.tsx:58]: `rem`instead `px`

File: views/home/hero/index.tsx [hunk 2]
```diff
@@ -39,19 +40,29 @@ const HeroSection: FC = () => {
...
+              fontStyle="normal"
+              fontWeight="500"
+              lineHeight="20px"
```
Comment (KipandaJr) [inline views/home/hero/index.tsx:61]: `rem`instead `px`

File: views/home/hero/index.tsx [hunk 3]
```diff
@@ -65,22 +76,23 @@ const HeroSection: FC = () => {
           </Typography>
           <Link href="/buy">
             <Button
-              py="1.5rem"
-              px="2.5rem"
-              fontWeight="700"
+              width="8.75rem"
+              py="1rem"
+              px="1.2rem"
+              fontSize="14px"
```
Comment (KipandaJr) [inline views/home/hero/index.tsx:82]: `rem`instead `px`

**Review by KipandaJr — CHANGES_REQUESTED** [review]
image
 check the mobile version

image

image

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**